Source/WebCore/platform/text/TextFlags.cpp

```diff
@@ -278,13 +264,60 @@ FeaturesMap computeFeatureSettingsFromVariants(const FontVariantSettings& varian
         break;
     }
 
+    // We don't do anything if "normal".
     if (!variantSettings.alternates.isNormal()) {
         auto values = variantSettings.alternates.values();
         if (values.historicalForms)
             features.set(fontFeatureTag("hist"), 1);
         
-        // TODO: handle other tags.
-        // https://bugs.webkit.org/show_bug.cgi?id=246121
+        if (fontFeatureValues) {
+            auto lookupTags = [](const auto& name, const auto& tags) -> Vector<int> {
```

Comment:
It’s really inefficient that we have to copy a vector to return the vector that is sitting in the hash table. We can instead use `Span<const int>` for the return value and then we won’t be allocating and destroying temporary vectors all the time.